front/src/common/Tabs.tsx

```diff
@@ -0,0 +1,49 @@
+import React, { useState } from 'react';
+import nextId from 'react-id-generator';
+
+
+type TabProps = {
+  label: string;
+  content: React.ReactNode;
+};
+
+type TabsProps = {
+  tabs: TabProps[];
+};
+
+const Tab = ({ label, content }: TabProps) => {
+  return (
+    <div className="tab-pane active" aria-labelledby={label}>
+      {content}
+    </div>
+  );
+};
+
+const Tabs = ({ tabs }: TabsProps) => {
+  const [activeTabIndex, setActiveTabIndex] = useState(0);
+
+  const handleTabClick = (index: number) => {
+    setActiveTabIndex(index);
+  };
+
+  return (
+    <div className="tabs">
+      <div className="d-flex mt-2 mb-3">
+        {tabs.map((tab, index) => (
+          <div
+            className={`tab pb-3 mr-4 ${index === activeTabIndex ? 'active' : ''}`}
+            key={nextId()}
```

Comment:
If possible I would avoid using nextId. it’s not always reliable. we’ve had bug because of it.

- Prefer using what you have available (if you have tab.id, tab.name, tab.label use it).
- you can be creative to make sure the `key` is unique ``` `${tab.id}-${tab.label}-${index}` ```
- using `index` is also fine as long as the array is not filtered/reordered, and the items don’t change

Reply:
Hi ! I don't agree with you @anisometropie let's talk about it on slack ;-)